openstack/swift/templates/seeds.yaml

```diff
@@ -60,33 +55,19 @@ spec:
     - name: swift_dispersion
       description: 'Swift Dispersion'
       password: {{ $.Values.dispersion_password }}
-    - name: ironic
-      roles:
-      - project: service
-        role: swiftreseller
```

Comment:
This looks wrong, but it's okay because the Ironic seed [assigns this role](https://github.com/sapcc/helm-charts/blob/8d779fa475dddb591dc2bbd4aa94a12498c48087/openstack/ironic/templates/seed.yaml#L218-L219) and has the correct dependency on this seed.

Reply:
agree